src/libutil/hash_test.cpp

```diff
@@ -191,7 +130,41 @@ getargs (int argc, char *argv[])
     }
 }
 
-
+/*
+ * PCG Random Number Generation
+ *
+ * Copyright 2014 Melissa O'Neill <oneill@pcg-random.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For additional information about the PCG random number generation scheme,
+ * including its license and other licensing options, visit
+ *
+ *     http://www.pcg-random.org
+ */
+
+struct pcg32_random_t {                     // Internals are *Private*.
```

Comment:
Am I right in understanding that there is no way to seed this? So what's the point? If only used in this unit test, which is obviously not performance-critical (not in the generating of the strings, anyway), why bother with this versus just using std random numbers?

Reply:
Yeah I simplified it since its just used for the test. It would be easy to add a seeding constructor if we wanted it elsewhere.

I can switch to `std::random` here. The actual values don't matter.